data/core.yaml

```diff
@@ -2110,6 +2114,8 @@ en:
         title: "Display options"
         background: "Toggle background imagery pane"
         background_switch: "Switch back to last background"
+        background_previous: "Switch to previous background in list"
```

Comment:
Consider removing 'in list' from this description- I think 'switch to previous background' is perfectly adequate